packages/ai-jsx/src/batteries/sidekick/platform/sidekick.tsx

```diff
@@ -108,10 +106,16 @@ type OutputFormatSidekickProps = MergeExclusive<
 export type SidekickProps = UniversalSidekickProps & OutputFormatSidekickProps;
 
 export function Sidekick(props: SidekickProps) {
+  const model = props.model ?? 'gpt-4-32k';
```

Comment:
Why does `Sidekick` do this at all instead of just using whatever is set on context?

Reply:
Good question. The main reason I wanted it was so we could do this error message, but I guess this error message is redundant with the one `UseTools` will throw. (`UseTools`' message is more generic and less actionable/useful, but we could fix that with an error boundary.)